server/pkg/asset/asset_test.go

```diff
@@ -37,6 +39,7 @@ func TestAsset_AssetType(t *testing.T) {
 	assert.Equal(t, wantPreviewType, got.PreviewType())
 	assert.Equal(t, &File{}, got.File())
 	assert.Equal(t, "yyy", got.Hash())
+	assert.Equal(t, thid, got.Thread())
 }
```

Comment:
this test function has not good name: `TestAsset_AssetType` because it no longer tests only AssetType method, but it tests all getter methods. so rename it, e.g. `TestAsset` `TestAsset_Getters`